camel/storages/graph_storages/graph_element.py

```diff
@@ -0,0 +1,69 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+from __future__ import annotations
+
+from typing import List, Union
+
+from pydantic import BaseModel, Field
+from unstructured.documents.elements import Element
+
+
+class Node(BaseModel):
+    r"""Represents a node in a graph with associated properties.
+
+    Attributes:
+        id (Union[str, int]): A unique identifier for the node.
+        type (str): The type or label of the node, default is "Node".
+        properties (dict): Additional properties and metadata associated with
+            the node.
+    """
+    id: Union[str, int]
+    type: str = "Node"
```

Comment:
Why we need to have this `"Node"` type in the `Node` class?

Reply:
User may need to set label to their node, this label will be passed into neo4j, `Node` is just the default naming

Reply:
Then maybe rename `type` to `label` and set the default value to None or empty string?

Reply:
Then maybe rename `type` (type is one of Python internal function we should avoid use it as the name of customized attribute or method) to `label` and change the default value to None or empty string